showcase/src/app/welcome/pages/reactive-form-errors/components/card/card.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, HostBinding, Input } from "@angular/core";
+
+@Component({
+	selector: "app-card",
+	templateUrl: "./card.component.html",
+	styleUrls: ["./card.component.scss"]
+})
+export class CardComponent {
+	@HostBinding("class.app-color-primary")
+	public primaryColor!: boolean;
+	@HostBinding("class.app-color-accent")
+	public accentColor!: boolean;
+	@HostBinding("class.app-color-warning")
+	public warningColor!: boolean;
+	@HostBinding("class.app-color-success")
+	public successColor!: boolean;
+
+	@Input()
+	/* tslint:disable-next-line:max-union-size */
```

Comment:
This TSLint violation is really easy to solve, you can create an alias type to contain the different colors. Could you fix this? That way we keep less things on our backlog ;)